projects/1.20/assets/blood-and-madness/bloodandmadness/lang/zh_cn.json

```
@@ -0,0 +1,246 @@
+{
+  "key.trick": "变换武器",
+  "key.reload": "重载",
+  "itemGroup.bloodborneblocks": "血源诅咒|方块",
+  "itemGroup.bloodborneweapons": "血源诅咒|武器",
+  "itemGroup.bloodborneentities": "血源诅咒|实体",
+  "itemGroup.bloodbornedecorations": "血源诅咒|装饰物",
+  "itemGroup.bloodborneresources": "血源诅咒|资源",
+  "itemGroup.bloodbornedebug": "血源诅咒|Debug",
+ 
+  "effect.bloodandmadness.trapped": "被困住!",

```

```suggestion
  "effect.bloodandmadness.trapped": "被困住！",
```

标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。